src/System.Threading.Tasks/src/System.Threading.Tasks.csproj

```diff
@@ -20,7 +20,7 @@
   <ItemGroup Condition="'$(TargetGroup)' == ''">
     <Compile Include="System\Threading\Tasks\TaskExtensions.CoreCLR.cs" />
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
-      <OSGroup>Windows_NT</OSGroup>
+      <OSGroup>$(TestOsTarget)</OSGroup>
```

Comment:
TestOSTarget shouldn't be used in the product builds. Is there are reason you needed to update this?